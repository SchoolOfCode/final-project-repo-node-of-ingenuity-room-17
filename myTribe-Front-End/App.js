import { StatusBar } from 'expo-status-bar';
import { StyleSheet, View } from 'react-native';
import { useState } from 'react';
import Family from './screens/Family';

export default function App() {
	return <Family />;
}

const styles = StyleSheet.create({
	mainView: {
		height: '100%',
		width: '100%',
	},
});
