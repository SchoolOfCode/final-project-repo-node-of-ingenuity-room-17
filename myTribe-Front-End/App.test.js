import React from "react";
import { View } from "react-native";
import renderer from "react-test-renderer";

import App from "./App";

//Test to check if the app is being rendered
describe("App created", () => {
  it("has 1 child", () => {
    const tree = renderer.create(<App />).toJSON();
    expect(tree.children.length).toBe(1);
  });

  //Test to check if what is being rendered in 'app' matches the snapshop in "_snapshot" file
  it("renders correctly", () => {
    const tree = renderer.create(<App />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
