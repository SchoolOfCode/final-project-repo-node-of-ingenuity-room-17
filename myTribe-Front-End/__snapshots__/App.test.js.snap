// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App created renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#fff",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text>
    "Hello World!"
  </Text>
</View>
`;
